src/meta/src/barrier/progress.rs

```diff
@@ -371,6 +385,17 @@ impl CreateMviewProgressTracker {
         self.actor_map.retain(|_, table_id| *table_id != id);
     }
 
+    /// Notify all tracked commands that error encountered and clear them.
+    pub fn abort_all(&mut self, err: &MetaError) {
```

Comment:
The progress tracker will be recreated in `recover_background_mv_progress`, so this method is not needed.